.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,5 +44,6 @@ Python packages changed:
 
 <!--- Put an `x` in all the boxes that apply. -->
 
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
+- [ ] My change does not require any changes to the documentation.
+- [ ] My change requires an update to the documentation which does not happen in this PR. <!--- Elaborate below on when the update will happen. -->
+- [ ] My change requires an update to the documentation which is part of this PR.

```

Comment:
For the documentation section, we should have a third option as well. Lots of PR as bug fixes and may not always have a doc change.

 My change requires a change to the documentation.
 I have updated the documentation accordingly.
 **My change doesn't change any documentation**

Reply:
Isn't that the first option? "My change does not require any changes to the documentation."

As far as I can tell there's only 3 paths: no changes, changes required but I'm not doing them in this PR, changes required and I did them.

Reply:
sorry, I probably missed this.

Reply:
No problem! I totally agree and didn't like the way I had originally written it. That's basically the motivation for the PR, so I'm glad we're on the same page 🙂